Source/SwiftLintFramework/Rules/Performance/EmptyCountRule.swift

```diff
@@ -15,19 +15,26 @@ public struct EmptyCountRule: ConfigurationProviderRule, OptInRule, AutomaticTes
             "[Int]().isEmpty\n",
             "[Int]().count > 1\n",
             "[Int]().count == 1\n",
+            "[Int]().count == 0xff\n",
+            "[Int]().count == 0b01\n",
+            "[Int]().count == 0o07\n",
             "discount == 0\n",
             "order.discount == 0\n"
         ],
         triggeringExamples: [
             "[Int]().↓count == 0\n",
             "[Int]().↓count > 0\n",
             "[Int]().↓count != 0\n",
+            "[Int]().↓count == 0x0\n",
+            "[Int]().↓count == 0x00_00\n",
+            "[Int]().↓count == 0b00\n",
+            "[Int]().↓count == 0o00\n",
             "↓count == 0\n"
         ]
     )
 
     public func validate(file: File) -> [StyleViolation] {
-        let pattern = "\\bcount\\s*(==|!=|<|<=|>|>=)\\s*0"
+        let pattern = "\\bcount\\s*(==|!=|<|<=|>|>=)\\s*0(\\b|([box][0_]+\\b){1})"
```

Comment:
I thought `box` was a special keyword or something, took me a while to realize it's just the three letters that are allowed 😅

Reply:
Hahaha, same